### ISSUE TITLE
[RFC] Should we deprecate this extension?

### ISSUE THREAD

**Issue by Bladrak** [thumbor-community/core#20]
Doesn't seem to be used much, what do you think?

**Comment by gi11es** [comment]
We use it in production for Wikipedia, as a way to bring in our custom handlers:

https://github.com/wikimedia/mediawiki-vagrant/blob/ed2623845f47c38569825cd1ce95f07ac2bb6372/puppet/modules/thumbor/templates/20-thumbor-wikimedia.conf.erb#L5-L8

As well as to override the "app":

https://github.com/wikimedia/mediawiki-vagrant/blob/ed2623845f47c38569825cd1ce95f07ac2bb6372/puppet/modules/thumbor/templates/20-thumbor-wikimedia.conf.erb#L92-L93

**Comment by Bladrak** [comment]
Oh ok, didn't know about that. Should we plan for a python 3 / thumbor 7 upgrade then?

**Comment by gi11es** [comment]
Yes. If nobody gets around to it, we will have to handle it. Currently we're waiting for Thumbor 7 to be stable before we start working on our migration effort.

**Comment by mpdude** [comment]
Thumbor 7 seems to be a prerequisite for using the new `arm64` platform, so 👀

**Comment by Bladrak** [comment]
All right, closing this in favour of #21 then :-)
